Capstone/Detail View/DetailsViewController.m

```diff
@@ -153,6 +164,20 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
     LongDetailsViewController *longDVC = [segue destinationViewController];
     longDVC.college = collegeToPass;
 }
+
+- (CLLocationCoordinate2D)getLocation {
+    self->locationManager = [[CLLocationManager alloc] init];
+    self->locationManager.delegate = self;
```

Comment:
I suggest moving these two initialization statements into the viewDidLoad function!

Reply:
Resolved